typed-racket-lib/typed-racket/base-env/prims-contract.rkt

```diff
@@ -152,7 +153,8 @@
    (pattern oc:opaque-clause #:attr spec
      #`(require/opaque-type oc.ty oc.pred #,lib . oc.opt))
    (pattern (~var strc (struct-clause legacy)) #:attr spec
-     #`(require-typed-struct strc.nm (strc.body ...) strc.constructor-parts ...
+     #`(require-typed-struct strc.nm (strc.tvar ...)
```

Comment:
There is a default for this syntax class attribute (`strc.tvar`) that is the empty list.